diff --git a/YEAR/DAY/Cargo.toml b/YEAR/DAY/Cargo.toml
new file mode 100644
index 0000000..cff40c2
--- /dev/null
+++ b/YEAR/DAY/Cargo.toml
@@ -0,0 +1,12 @@
+[package]
+name = "advent-of-code-YEAR-DAY"
+version = "0.1.0"
+authors = ["Tobias Happ <tobias.happ@gmx.de>"]
+edition = "2018"
+
+[dependencies]
+error = { path = "../../lib/rust/error" }
+file = { path = "../../lib/rust/file" }
+
+[dev-dependencies]
+hamcrest2 = "0.3.0"
diff --git a/YEAR/DAY/src/main.rs b/YEAR/DAY/src/main.rs
new file mode 100644
index 0000000..955902b
--- /dev/null
+++ b/YEAR/DAY/src/main.rs
@@ -0,0 +1,13 @@
+#![deny(warnings)]
+#![allow(unused_imports)]
+
+// necessary for intellij support
+#[cfg(test)]
+#[macro_use]
+extern crate hamcrest2;
+
+fn main() {}
+
+#[cfg(test)]
+mod tests {
+    use hamcrest2::prelude::*;
+
+    use super::*;
+}
